src/modules/budget/application/Budget/Shared/Services/BudgetImportService.cs

```diff
@@ -62,20 +62,20 @@ public async Task<GetImportResult> Import(IFormFile file)
 		await using (var streamWriter = new StreamWriter(memoryStream))
 		await using (var csvWriter = new CsvWriter(streamWriter, CultureInfo.InvariantCulture))
 		{
-			this.csvService.WriteRecordsToMemoryStream(budgetInfos.Result.BudgetsList, csvWriter);
+			this.csvService.WriteRecordsToMemoryStream(budgetInfos.Result.BudgetsErrorsList, csvWriter);
 			memoryStream.Position = 0;
 
 			await this.blobStorageService.UploadToBlobStorage(memoryStream, fileName);
 		}
 
 		string uri = await this.blobStorageService.GenerateLinkToDownload(fileName);
 
-		var download = await this.blobStorageService.DownloadFromBlobStorage(fileName);
-		using var reader = new StreamReader(download.Content);
-		using var csvReader = new CsvReader(reader, csvConfig);
-		await csvReader.ReadAsync();
-		var budgetsToImport = csvReader.GetRecords<GetBudgetTransferInfo>();
-		var budgetsAggregateList = await this.budgetDataService.ConvertBudgetsFromCsvToBudgetAggregate(budgetsToImport, csvConfig);
+		var budgetsAggregateList = await this.budgetDataService.ConvertBudgetsToBudgetAggregate(budgetInfos.Result.BudgetsList);
+
+		if (budgetInfos.Result.BudgetsErrorsList.Count == 0)
```

Comment:
if(!budgetInfos.Result.BudgetsErrorsList.Any())
```suggestion
		if (budgetInfos.Result.BudgetsErrorsList.Count == 0)
```